webextension/_locales/en_US/messages.json

```diff
@@ -71,6 +71,12 @@
     "message": "Description"
   },
 
+  "titleInitialPromptNoContext": {
+    "message": "Are you satisfied with how this website works in Firefox?"
+  },
+  "textInitialPromptNoContext": {
```

Comment:
This can be `textInitialPromptBasic`, and then `textInitialPromptFriendly` can be the "detect issues, etc" one.